src/adapt.ts

```diff
@@ -438,7 +434,7 @@ export function adaptOutputUtil(
     rawResult: string,
     outputSpecs: starknet.Argument[],
     abi: starknet.Abi
-): StringMap {
+): unknown {
```

Comment:
This seems like a very disruptive change. As much as I don't like `any`, but it would be less disruptive.

Reply:
https://github.com/0xSpaceShard/starknet-hardhat-plugin/pull/368/commits/ad4f8352e5a1069ca34c86d09f56ade94f27448c